### PR TITLE
feat: deployment for arbitrum testnet

### PR DESCRIPTION
**Pull request by leoslr**
## Description of the changes

Deployment on arbitrum testnet of the following smart contracts :
 - ChainlinkConversionPath
 - ERC20FeeProxy
 - ERC20ConversionProxy
 - ETHFeeProxy
 - ETHProxy

Arbitrum testnet native currency (ARETH) added to the currency list.

### CODE REVIEW

File: packages/currency/src/native.ts [hunk 1]
```diff
@@ -75,6 +75,12 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Near Testnet',
       network: 'aurora-testnet',
     },
+    {
+        symbol: 'ARETH',
```
Comment (KolevDarko) [inline packages/currency/src/native.ts:79]: we usually put testnet in the symbol as well so we can differentiate between testnet and mainnet, like on Near

**Review by KolevDarko — APPROVED** [review]

File: packages/currency/src/native.ts [hunk 2]
```diff
@@ -75,6 +75,12 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
...
+        decimals: 18,
+        name: 'Arbitrum Testnet',
+        network: 'arbitrumtestnet'
```
Comment (kevindavee) [inline packages/currency/src/native.ts:82]: Adding a dash seems to increase its readability (`arbitrum-testnet`). Is it possible?

**Review by kevindavee — APPROVED** [review]
Minor comment. But congrats on your PR!

File: packages/currency/src/native.ts (hunk 2 above)
Comment (leoslr) [reply packages/currency/src/native.ts:82]: Done !

File: packages/currency/src/native.ts (hunk 1 above)
Comment (KolevDarko) [reply packages/currency/src/native.ts:79]: Had no idea, congrats on the first PR though  :)

File: packages/smart-contracts/src/lib/artifacts/ChainlinkConversionPath/index.ts [hunk 3]
```diff
@@ -45,6 +45,10 @@ export const chainlinkConversionPath = new ContractArtifact<ChainlinkConversionP
           address: '0xEEc4790306C43DC00cebbE4D0c36Fadf8634B533',
           creationBlockNumber: 10141032,
         },
+        arbitrumtestnet: {
```
Comment (kevindavee) [inline packages/smart-contracts/src/lib/artifacts/ChainlinkConversionPath/index.ts:48]: You need to use `''` in order to use `-` as object key name.
```
'arbitrum-testnet': {}
```

Comment (leoslr) [reply packages/smart-contracts/src/lib/artifacts/ChainlinkConversionPath/index.ts]: Thanks for the tip, it's corrected

**Review by Dadogg80 — APPROVED** [review]

File: packages/currency/src/native.ts (hunk 2 above)
Comment (bertux) [reply packages/currency/src/native.ts:82]: I don't see it done here, but it's done in all other places.

Comment (bertux) [reply packages/currency/src/native.ts:82]: and indentation should be 2 spaces and not 4 as in the other blocks.

**Review by bertux — APPROVED** [review]

**Pull request merged**